### DISCUSSION TITLE
help me plz

### DISCUSSION THREAD

**Discussion by Funemexican** [KadeArchive/Kade-Engine#1831] [Q&A]
so i was downloading kade engine and i was putting lime test windows and it said lime tool kit not found i now i dont know what to do i looked back to the video and i downloaded every thing so sombody help me plz

**Comment by Gostar64** [comment]
I'm pretty sure you have to get the haxe toolkit and haxefilxel (if you don't have them already) then you put haxelib install lime (something like that)

If that doesn't work, I have no idea what you can do besides retrying everything using the building page as a guide
